recipe_scrapers/panelinha.py

```diff
@@ -26,10 +30,24 @@ def instructions(self):
         instructions = self.soup.find(
             "h4", string="Modo de preparo"
         ).nextSibling.findAll("li")
-
-        return "\n".join(
-            [normalize_string(instruction.get_text()) for instruction in instructions]
-        )
+        instructions = [
+            normalize_string(instruction.get_text()) for instruction in instructions
+        ]
+
+        # Some recipes pages have a different html structure.
+        if not instructions:
+            instructions = self.soup.find(
+                "h4", string="Modo de preparo"
+            ).nextSibling.p.contents
+            instructions = filter(lambda x: isinstance(x, str), instructions)
```

Comment:
nit: This filter could (seemingly) be easy to put into the comprehension below for performance gains.

Reply:
Thanks @bfcarpio. 
I just found out I can use `.strings` on L41 instead of `.contents`.
That way, it is not needed to filter for strings to remove the `<br/>` elements.